src/modules/blocks/ticket/container-content/attendee-collection/style.pcss

```diff
@@ -0,0 +1,74 @@
+.tribe-editor__ticket__attendee-collection {
+	margin-top: 20px;
+
+	.tribe-editor__accordion__row {
```

Comment:
I think you should be able to add classes to the accordion row as well? Best to do that and use that to target element to style.

Reply:
Could totally do that, was just up against a time crunch there, did not go too much further than getting things functional and looking roughly like the intended interface designed.